src/network/networkbase.h

```diff
@@ -0,0 +1,173 @@
+#ifndef SERVERDATA_H
+#define SERVERDATA_H
+
+#include <QObject>
+#include <QString>
+#include <QDateTime>
+#include <QJsonArray>
+#include <QJsonObject>
+#include <QJsonDocument>
+#include <QUdpSocket>
+#include <QTcpServer>
+
+#define CUSTOM_DEBUG
+
+// Custom types
+typedef quint16 PortNumType;
+typedef qint64  SizeType;
+
+// Default values for common parameters
+const QString     gDefaultAppLabel         = "PairStorm";
+const QString     gDefaultOrgLabel         = "cpp-lv425";
```

Comment:
this information available in QSettings:
QSettings s;
QString name = s.value("applicationName").toString();

Reply:
Thank's, now I receive it from QSettings.